cloudFerry.py

```diff
@@ -202,5 +262,33 @@ def create_filters(name_config=None, filter_folder=DEFAULT_FILTERS_FILES,
     make_filters.make(filter_folder, images_date)
 
 
+def init(args):
+    install_folder = resource_filename(Requirement.parse("CloudFerry"), './')
+    ignore_py = shutil.ignore_patterns('*.py', '*.pyc')
+
+    def copy(folder):
+        shutil.copytree("%s/%s" % (install_folder, folder), "./%s" % folder,
+                        False, ignore_py)
+
+    copy('configs')
+    copy('scenario')
+    copy('templates')
+
+
+commands.append(Command('init', init))
```

Comment:
please group all `commands.append()` calls in one place so that they are not scattered around the file.